src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -1547,4 +1649,13 @@ public sealed partial class PublishingOptions
 
         public string? Publisher { get { throw null; } set { } }
     }
+}
+
+namespace Aspire.Hosting.Utils
```

Comment:
This feels like a bad namespace.

Reply:
@DamianEdwards Can you fix this one?

Reply:
https://github.com/dotnet/aspire/pull/7464